clients/src/test/java/org/apache/kafka/common/requests/MetadataRequestTest.java

```diff
@@ -74,20 +75,30 @@ public void testMetadataRequestVersion() {
     @Test
     public void testTopicIdAndNullTopicNameRequests() {
         // Construct invalid MetadataRequestTopics. We will build each one separately and ensure the error is thrown.
-        List<MetadataRequestData.MetadataRequestTopic> topics = Arrays.asList(
+        List<MetadataRequestData.MetadataRequestTopic> invalidTopics = Arrays.asList(
                 new MetadataRequestData.MetadataRequestTopic().setName(null).setTopicId(Uuid.randomUuid()),
                 new MetadataRequestData.MetadataRequestTopic().setName(null),
                 new MetadataRequestData.MetadataRequestTopic().setTopicId(Uuid.randomUuid()),
                 new MetadataRequestData.MetadataRequestTopic().setName("topic").setTopicId(Uuid.randomUuid()));
 
+        List<MetadataRequestData.MetadataRequestTopic> validTopics = Arrays.asList(
```

Comment:
nit: Would it make sense to split this test into two tests, one for the valid cases and one for the invalid cases? I think that it would be easier to read.

Reply:
done